pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -291,7 +291,7 @@ type PostFilterPlugin interface {
 	PostFilter(ctx context.Context, state *CycleState, pod *v1.Pod, filteredNodeStatusMap NodeToStatusMap) (*PostFilterResult, *Status)
 }
 
-// PreScorePlugin is an interface for Pre-score plugin. Pre-score is an
+// PreScorePlugin is an interface for "PreScore" plugin. Pre-score is an
```

Comment:
Do you want to fix the second `Pre-score` on this line as well?

Reply:
> Do you want to fix the second `Pre-score` on this line as well?
yes, fixed it.